pyuvdata/uvdata/mir_parser.py

```diff
@@ -263,15 +269,22 @@ def __init__(self, filepath, load_vis=False, load_raw=False, load_auto=False):
         self.bl_filter = np.ones(self.bl_read.shape, dtype=np.bool_)
         self.sp_filter = np.ones(self.sp_read.shape, dtype=np.bool_)
         self.we_filter = np.ones(self.we_read.shape, dtype=np.bool_)
-        self.ac_filter = np.ones(self.ac_read.shape, dtype=np.bool_)
 
         self.in_data = self.in_read
         self.eng_data = self.eng_read
         self.bl_data = self.bl_read
         self.sp_data = self.sp_read
         self.codes_data = self.codes_read
         self.we_data = self.we_read
-        self.ac_data = self.ac_read
+
+        if read_auto:
+            self.ac_read = self.scan_auto_data(filepath)
+            self.ac_filter = np.ones(self.ac_read.shape, dtype=np.bool_)
+            self.ac_data = self.ac_read
+        else:
+            # Check for load_auto=True.
```

Comment:
I think I'd like to preserve the pattern used with, for example, `vis_data`, `raw_data`, and `auto_data`, and set the three attributes to `None` when we don't load them (which more generally mimics the behavior of `UVData` objects, and gets around needing to use `has_attr`).

Reply:
Got it. I've defaulted all of the `ac_*` to None.